api/src/main/java/graphql/nadel/dsl/UnderlyingServiceHydration.java

```diff
@@ -51,13 +53,18 @@ public UnderlyingServiceHydration(SourceLocation sourceLocation,
         this.objectIndexed = objectIndexed;
         this.batched = batched;
         this.batchSize = batchSize;
+        this.timeout = timeout;
         this.syntheticField = syntheticField;
     }
 
     public Integer getBatchSize() {
         return batchSize;
     }
 
+    public Integer getTimeout() {
```

Comment:
Is this ever nullable? If not then can we just return primitive `int`?

No big deal though, since this class is barely touched.

Reply:
changed